compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -7006,6 +7012,27 @@ private BType checkInvocationArgs(BLangInvocation iExpr, List<BType> paramTypes,
         }
     }
 
+    private void addToIncRecords(BVarSymbol param, List<BRecordType> incRecords) {
```

Comment:
We can say 
```suggestion
    private void populateIncludedRecordParams(BVarSymbol param, List<BRecordType> incRecords) {
```

I don't see why we need to shorten `Included` to `Inc`.

Reply:
Changed, 
`Inc` is used in order to align with other usages of this file.